lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
I wonder anyway - how often does this happen? We can of course make everything everywhere type *really* *really* slow and wait forever in between to avoid seeing suprises, but the question is what we gain in the long run.